gips/inventory/dbinv/api.py

```diff
@@ -226,13 +226,14 @@ def update_or_add_asset(driver, asset, tile, date, sensor, name):
         'asset':  asset,
         'tile':   tile,
         'date':   date,
+        'status': models.Status.objects.get(status=status),
     }
     update_vals = {'sensor': sensor, 'name': name}
     (asset, created) = models.Asset.objects.update_or_create(defaults=update_vals, **query_vals)
     return asset # in case the user needs it
 
 
-def update_or_add_product(driver, product, tile, date, sensor, name):
+def update_or_add_product(driver, product, tile, date, sensor, name, status):
```

Comment:
just to tag it for git's review tracker thing - Suggest adding a default of 'complete' to to the status arg for this fn